api/src/opentrons/config/types.py

```diff
@@ -131,22 +131,34 @@ class CapacitivePassSettings:
 
 @dataclass(frozen=True)
 class ZSenseSettings:
-    point: Offset
     pass_settings: CapacitivePassSettings
 
 
 @dataclass(frozen=True)
 class EdgeSenseSettings:
-    plus_x_pos: Offset
-    minus_x_pos: Offset
-    plus_y_pos: Offset
-    minus_y_pos: Offset
     overrun_tolerance_mm: float
     early_sense_tolerance_mm: float
     pass_settings: CapacitivePassSettings
     search_initial_tolerance_mm: float
     search_iteration_limit: int
-    nominal_center: Offset
+
+    def __init__(
+        self,
+        overrun_tolerance_mm: float,
+        early_sense_tolerance_mm: float,
+        pass_settings: CapacitivePassSettings,
+        search_initial_tolerance_mm: float,
+        search_iteration_limit: int,
+    ) -> None:
+        if not (overrun_tolerance_mm <= pass_settings.max_overrun_distance_mm):
```

Comment:
can probably say 

```suggestion
        if (overrun_tolerance_mm > pass_settings.max_overrun_distance_mm):
```

for a little more readability

Reply:
ha good call!